### ISSUE TITLE
Several serious container errors are being reported at the DEBUG level

### ISSUE THREAD

**Issue by bcbrock** [hyperledger-archives/fabric#653]
There are several log messages related to container handling that I think should be promoted from DEBUG level to a higher level, as these represent errors from the user's perspective. This will make it easier for users (like me) to figure out what's going on, as these messages are normally not printed.

Here are a few examples:

```
12:38:25.785 [container] build -> DEBU 029 Error building Peer container: Error: image bcbrock/golang:latest not found
12:38:25.800 [container] start -> DEBU 031 start-container does not exist, attempting to create API error (404): no such id: dev-vp1-60caf3632e0f43a5a7f14f27115bfaf84f76ee59d3490bbb337703e098ada77fc7b7bddd093f354e8ef89f644330131e5799c966503070b94eb1a83cbc9dc10c
12:38:25.803 [container] start -> DEBU 033 start-could not recreate container no such image
12:57:11.180 [chaincode] LaunchChaincode -> DEBU 09c launchAndWaitForRegister failed Error starting container: no such image
```

**Comment by muralisrini** [comment]
@bcbrock : I'll take a look at this.

**Comment by srderson** [comment]
@muralisrini adding a help wanted tag in case someone wants to pick this up. Let me know if you already have a fix.

**Comment by juliancarrivick-ibm** [comment]
@bcbrock I've replaced a heap of errors which were being reported at debug in `handler.go`. The first three that you listed in the OP seems to have been fixed in PR #738.

I've also done a search for debug statements that contain the word fail and adjusted some of them too.
